Assets/Scripts/KatamariMovement.cs

```diff
@@ -31,30 +31,54 @@ public class KatamariMovement : MonoBehaviour
 
     public bool isGolfHitMode = true;
 
-    public GameObject power_bar;
+    public GameObject powerBar;
 
     private bool go_up;
     public float power;
     public float time_modifier;
     private float angle_timer = 0;
-    public bool power_bar_active = true;
+    public bool powerBarActive = true;
```

Comment:
Is `powerBarActive` needed anymore? I think we may be able to reference the state machine instead.

Reply:
i think we dont need them anymore

Reply:
Removed in follow on PR